universal-registrar/test/post.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/env bash
 
+# path to keyring
 key_path="universal-registrar/test/keyring.json"
 
+# Used when testing on uniregistrar.io, may break testing of dockerimage
+method="?method=dyne"
+
+# Set endpoint to uni-registrar
+# user "http://localhost:3000/1.0/" to test in localhost
+endpoint="https://uniregistrar.io/1.0/"
+
+
 call_api() {
     curl -X 'POST' -o output.json \
-	 "http://localhost:3000/1.0/${1}" \
+	 "${endpoint}${1}${method}" \
```

Comment:
A super small better approach is to hardcode the `$` in here, this make the string concatenation future proof... if in the future you want to add more options, then you don't need to refactor this!

Reply:
what do you mean by hardcoding the `$`? To write directly there the value `https://uniregistrar.io/1.0/${1}?method=dyne`?